### PR TITLE
feat: add test reporting to component generation

### PR DESCRIPTION
**Pull request by devpow112**
https://desire2learn.atlassian.net/browse/QE-264

This adds the configuration pieces for test reporting and fixes a few other minor inconsistencies.

### CODE REVIEW

File: src/generators/test-unit-axe/templates/configured/_element.axe.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import '../<%= hyphenatedName %>.js';
 import { expect, fixture, html } from '@brightspace-ui/testing';
 
-describe('<%= className %>', () => {
+describe('<%= tagName %>', () => {
```
Comment (devpow112) [inline src/generators/test-unit-axe/templates/configured/_element.axe.js:4]: These didn't match what we tend to do in the wild and also didn't match out vdiff suite names. Changing to be all the same

File: src/generators/test-unit-axe/templates/configured/_element.test.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import '../<%= hyphenatedName %>.js';
 import { runConstructor } from '@brightspace-ui/testing';
 
-describe('<%= className %>', () => {
+describe('<%= tagName %>', () => {
```
Comment (devpow112) [inline src/generators/test-unit-axe/templates/configured/_element.test.js:4]: These didn't match what we tend to do in the wild and also didn't match out vdiff suite names. Changing to be all the same

File: README.md [hunk 3]
```diff
@@ -44,6 +45,13 @@ In order for the release workflow to automatically update the version, the repo
 
 Learn more in the [action docs](https://github.com/BrightspaceUI/actions/blob/main/docs/release-token.md).
 
+### Test Reporting
+
+In order test reports to be submitted, the repo needs to be configured with `AWS_*` tokens with the correct permissions.
```
Comment (dlockhart) [inline README.md:50]:
```suggestion
In order for test reports to be submitted, the repo needs to be configured with `AWS_*` tokens with the correct permissions.
```

File: src/create.js [hunk 4]
```diff
@@ -121,12 +122,45 @@ async function getComponentOptions() {
...
 			process.exit();
 		},
 	});
+
+	if (templateData.testReporting) {
+		const testReportingTemplateData = await prompts([
+			{
+				type: 'text',
+				name: 'testReportingTool',
+				message: 'What is your components tool based on the taxonomy in https://expanse.desire2learn.com/pages/source/source.html'
```
Comment (dlockhart) [inline src/create.js:145]:
```suggestion
				message: 'What is your component\'s tool based on the taxonomy in https://expanse.desire2learn.com/pages/source/source.html'
```

File: src/create.js [hunk 5]
```diff
@@ -121,12 +122,45 @@ async function getComponentOptions() {
...
+			}, {
+				type: 'text',
+				name: 'testReportingExperience',
+				message: 'What is your components experience based on the taxonomy in https://expanse.desire2learn.com/pages/source/source.html'
```
Comment (dlockhart) [inline src/create.js:149]:
```suggestion
				message: 'What is your component\'s experience based on the taxonomy in https://expanse.desire2learn.com/pages/source/source.html'
```

**Review by dlockhart — APPROVED** [review]

**Pull request merged**